pkg/registry/common/refresh/options.go

```diff
@@ -20,7 +20,7 @@ import "time"
 
 type configurable interface {
```

Comment:
Why an interface here?

Reply:
I did add this for support server/client but actually pkg refresh has the only client, so we can just remove this interface.